test/distributed/_tensor/test_math_ops.py

```diff
@@ -394,6 +394,26 @@ def _replicate_fn(name, module, device_mesh):
 
             self.assertEqual(x_local.grad, x_dist.grad.full_tensor())
 
+    @with_comms
+    def test_topk(self):
+        device_mesh = self.build_device_mesh()
+        placement_combs = [Shard(0), Shard(1), Shard(2), Replicate()]
+
+        comm_mode = CommDebugMode()
```

Comment:
nit: looks like comm_mode is not used in the test?

Reply:
good catch! updated